src/core/a-animation.js

```diff
@@ -391,7 +398,14 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
     }
     from[attribute] = parseProperty(from[attribute], schema[componentPropName]);
     to[attribute] = parseProperty(dataTo, schema[componentPropName]);
```

Comment:
If our `color` property type parser converted to `THREE.Color` I think everything would work out of the box. But I'm not sure if that would break things.